Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY: test
 test:
 	python -m pytest --version
-	python -m pytest -v test/
+	python -m pytest --xvfb-backend=xvfb -v test/
```

Comment:
If `xvfb` is not installed and a display is found, then `pytest-xvfb` will render widgets to your screen. This is not ideal for development. With this flag, you get an error prompting you to install `xvfb`.